src/py/Makefile

```diff
@@ -49,3 +49,23 @@ alembic-upgrade-head:
 alembic-autogenerate:
 	alembic revision --autogenerate -m "$(MESSAGE)" --rev-id $$(date +%Y%m%d_%H%M%S)
 
+.PHONY: init-db-cli
+init-db-cli:
```

Comment:
why not just add sqlalchemy-utils to one of the requirements file in the .happy directory?

Reply:
The Aspen CLI is installed into our flask docker image, and it's what needs sqlalchemy-utils installed to create/drop dev databases. The Happy CLI isn't meant to be run from containers for the most part -- it's executed by CI systems or engineers on their laptops.